### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by julie-jeancon-donon**

### CODE REVIEW

File: assets/styles/contact.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
+body {
+    overflow-x: hidden;
```
Comment (Guillaum6e) [inline assets/styles/contact.scss:2]: if it is meant to be of use in the whole website then put it in the global css,
if not then add a class pls

File: assets/styles/contact.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+form{
+  padding:20px;
```
Comment (Guillaum6e) [inline assets/styles/contact.scss:6]: same here

File: templates/contact/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    {{ encore_entry_link_tags('contactCss') }}
+    {{ encore_entry_link_tags('blurCss') }}
+{% endblock %}
+
+{% block body %}
+
+<main class="container">
+
+    {{ form_start(form,{'attr': {'class': 'bg-form'}}) }}
+    <p class="head_form">Un question, une suggestion? Contactez-nous!</p>
```
Comment (Guillaum6e) [inline templates/contact/index.html.twig:15]: Une question

**Review by Guillaum6e — APPROVED** [review]

File: src/Form/ContactType.php [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            ->add('firstname', TextType::class, [
+                'attr' => [
+                    'class' => 'form-control',
+                ],
+                'label' => 'Prénom',
+                'label_attr' => [
+                    'class' => 'form-label'
+                ],
+                ])
+            ->add('email', TextType::class, [
```
Comment (Benja-san) [inline src/Form/ContactType.php:45]: You can change TextType for EmailType

File: templates/contact/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    {{ encore_entry_link_tags('blurCss') }}
+{% endblock %}
+
+{% block body %}
+
+<div class="bg-overf">
+    <main class="container">
+
+        {{ form_start(form,{'attr': {'class': 'bg-form'}}) }}
+        <p class="head_form">Une question, une suggestion? Contactez-nous!</p>
```
Comment (Benja-san) [inline templates/contact/index.html.twig:16]: h1 would be a better option than a p

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**